src/ir_Mitsubishi.cpp

```diff
@@ -480,6 +480,16 @@ uint8_t IRMitsubishiAC::getWideVane(void) const {
   return _.WideVane;
 }
 
+/// Set the requested Left Vane (Vertical Swing) operation mode of the a/c unit.
+/// @param[in] position The position/mode to set the vane to.
+void IRMitsubishiAC::setVaneLeft(const uint8_t position) {
+  _.VaneLeft = std::min(position, kMitsubishiAcVaneAutoMove);  // bounds check
```

Comment:
[Question] Are all the possible values the same as for `setVane()` etc?

Reply:
I believe so. My real remote control has the same settings both left and right.